lib/ansible/modules/cloud/docker/_docker.py

```diff
@@ -25,7 +25,7 @@
 description:
   - This is the original Ansible module for managing the Docker container life cycle.
   - NOTE - Additional and newer modules are available. For the latest on orchestrating containers with Ansible
-    visit our Getting Started with Docker Guide at U(https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/guide_docker.rst).
+    visit our Getting Started with Docker Guide at U(https://docs.ansible.com/ansible/latest/scenario_guides/guide_docker.html).
```

Comment:
This should be :doc:`/scenario_guides/guide_docker` instead. This is preferable because it keeps the user's doc version when they are browsing online.